charts/backbeat-api/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
There's more useful metadata that can be added (refs to project pages, authors,...)